packages/base/src/mainview/ol.ts

```diff
@@ -0,0 +1,4 @@
+
+export * from 'ol';
+export { default as TileLayer } from 'ol/layer/Tile';
+export { default as XYZ } from 'ol/source/XYZ';
```

Comment:
This seems to mean OpenLayers is going to be the underlying library for interacting with maps?

Reply:
I opened an issue for this #7